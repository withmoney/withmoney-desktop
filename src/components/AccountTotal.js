import React from 'react';

const AccountTotal = () => (
  <div className="account-total">
    <div className="account-total__label">Your Account</div>
    <div className="account-total__money">$ 10,123.45</div>
  </div>
);

export default AccountTotal;
