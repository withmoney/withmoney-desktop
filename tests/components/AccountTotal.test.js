import renderer from 'react-test-renderer';
import AccountTotal from 'components/AccountTotal';

describe('AccountTotal', () => {
  let Component;

  beforeAll(() => {
    Component = global.setup(AccountTotal);
  });

  it('should render a component correctly', () => {
    const wrapper = renderer.create(Component());

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
