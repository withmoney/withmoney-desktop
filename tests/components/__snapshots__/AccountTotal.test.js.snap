// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountTotal should render a component correctly 1`] = `
<div
  className="account-total"
>
  <div
    className="account-total__label"
  >
    Your Account
  </div>
  <div
    className="account-total__money"
  >
    $ 10,123.45
  </div>
</div>
`;
